sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -865,23 +866,23 @@ qualifiedColTypeWithPositionList
     ;
 
 qualifiedColTypeWithPosition
-    : name=multipartIdentifier dataType (COMMENT comment=STRING)? colPosition?
+    : name=multipartIdentifier dataType (NOT NULL)? (COMMENT comment=STRING)? colPosition?
     ;
 
 colTypeList
     : colType (',' colType)*
     ;
 
 colType
-    : colName=errorCapturingIdentifier dataType (COMMENT STRING)?
+    : colName=errorCapturingIdentifier dataType (NOT NULL)? (COMMENT comment=STRING)?
```

Comment:
do we need to support specify `(NULL | (NOT NULL))?`

```sql
postgres=# create table a(a int null) ;
CREATE TABLE
```

Reply:
seems like not ANSI standard
![image](https://user-images.githubusercontent.com/3182036/71908953-e46ffd00-31a9-11ea-859f-09cef31c04e3.png)